README.md

```diff
@@ -1,16 +1,13 @@
 PERMON toolbox
 ================================
-PERMON (Parallel Efficient Robust Modular Object Numerical) is a software toolbox for quadratic programming (QP) based on [PETSc](http://www.mcs.anl.gov/petsc/). It also includes domain decomposition methods (FETI and Total FETI) as a specific QP implementation.
+PERMON (Parallel Efficient Robust Modular Object Numerical) is a software toolbox for the scalable (parallel) solution of  quadratic programming (QP) problems based on [PETSc](http://www.mcs.anl.gov/petsc/). It also includes domain decomposition methods (FETI and Total FETI) and Support Vector Machines (SVM) machine learning methods.
 
 Project homepage: <http://permon.vsb.cz>  
 
 PERMON contains following modules:
 
-* [PermonQP](http://permon.vsb.cz/permonqp.htm) - contains QP transformations and solvers
-* [PermonFLLOP](http://permon.vsb.cz/permonfllop.htm) - (FETI Light Layer On top of PETSc) implements (Total) FETI
-
-Support Vector Machines (SVMs) implementation [PermonSVM](http://permon.vsb.cz/permonsvm.htm) based on PermonQP can be found at <https://github.com/permon/permonsvm>.
-
+* [PermonQP](http://permon.vsb.cz/permonqp.htm) - solvers and routines for QP problems and (Total) [FETI](http://permon.vsb.cz/permonfllop.htm) domain decomposition methods
```

Comment:
```suggestion
* [PermonQP](http://permon.vsb.cz/permonqp.htm) - this repository
  - solvers and routines for solution of QP problems
  - constrained (Total) [FETI](http://permon.vsb.cz/permonfllop.htm) domain decomposition methods - solve contact problems with domain decomposition
```

Reply:
@jkruzik

Reply:
We could even drop "this repository".
I would suggest changing "domain decomposition methods - solve contact problems with domain decomposition" to
"domain decomposition methods able to solve contact problems"

Reply:
> We could even drop "this repository".

I would like to keep it for clarity as long as the PermonQP name is used. But maybe we could abandon this name for clarity. It's just PERMON, and PermonSVM is its extension for SVM.

> I would suggest changing "domain decomposition methods - solve contact problems with domain decomposition" to
"domain decomposition methods able to solve contact problems"

Updated suggestion:
```suggestion
* [PermonQP](http://permon.vsb.cz/permonqp.htm) - this repository
  - solvers and routines for solution of QP problems
  - constrained (Total) [FETI](http://permon.vsb.cz/permonfllop.htm) domain decomposition methods able to solve contact problems
```

Reply:
OK, keep it.
> It's just PERMON...

You can suggest it by mail to the others.